community/sql.gitattributes

```diff
@@ -1,4 +1,30 @@
-# Basic .gitattributes for sql files
+# Basic .gitattributes for SQL files
 
 *.sql linguist-detectable=true
 *.sql linguist-language=sql
+
+# Handle line endings automatically for files detected as text
+# and treat them as LF on check-in and CRLF when checked out.
+* text=auto eol=crlf
+
+# SQL files should be detected as text.
+*.sql text eol=crlf
+
+# Project and solution files
+*.sqlproj text eol=crlf
+*.sln text eol=crlf
```

Comment:
IDE-specific entries have their own files.
For example, for Visual Studio [link](https://github.com/gitattributes/gitattributes/blob/master/Global/VisualStudio.gitattributes).

This is the same organization scheme that `gitignore` is using, for example for Visual Studio [link](https://github.com/github/gitignore/blob/main/VisualStudio.gitignore)